templates/page/page.tpl.php

```diff
@@ -0,0 +1,88 @@
+<?php
```

Comment:
Why `templates/page/page.tpl.php`? I would've expected `templates/page.tpl.php` but if you have an alternate file structure in mind, I'm interested to know about it.